Apps/WebClient/src/ClientApp/src/components/dependentCard.vue

```diff
@@ -86,6 +86,7 @@ export default class DependentCardComponent extends Vue {
             .then((results) => {
                 if (results.resultStatus == ResultType.Success) {
                     this.labResults = results.resourcePayload;
+                    this.sortEntries();
```

Comment:
@ninosamson @tiagograf-AOT  It is good to sort the entries, but I don't see it as a part of the bug 9652.

Reply:
The test was broken due to a bug that i fixed in the same PR, whats the problem here?

Reply:
sounds good, thanks @tiagograf-AOT